test/nist_test.c

```diff
@@ -0,0 +1,263 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <time.h>
+#include <string.h>
+#include "../api.h"
+#include "../nist_params.h"
+#include "../xmss.h"
+#include "../params.h"
+
+#define XMSS_SIGNATURES 64
```

Comment:
Could you add a comment for what this represents?

Reply:
Yes, I just add the comment